script/server

```diff
@@ -24,6 +24,7 @@ if [ "$SWARM" = "inactive" ] ; then
     docker-compose down
   else
     echo "==> Starting the service..."
+    export CONFIG_FORCE_SSL=${CONFIG_FORCE_SSL:-false}
```

Comment:
Should the default be the same as in the docker-compose.yml file?

Reply:
It may make more sense when I add the Swarm bits - the compose sections are intended mainly to be used when testing, so disabling SSL makes sense here. The overall default should be to turn it on though as this is intended to be for production deploys.